packages/bitcore-node/src/config.ts

```diff
@@ -57,6 +57,7 @@ const Config = function(): ConfigType {
     dbHost: process.env.DB_HOST || '127.0.0.1',
     dbName: process.env.DB_NAME || 'bitcore',
     dbPort: process.env.DB_PORT || '27017',
+    asmLengthLimit: 1_000_000,
```

Comment:
This can definitely be set much lower without missing the primary use cases. With a lower limit, we'll have to be more thoughtful on the application side to gather info which might otherwise be simple mongo queries, but all OP_RETURN protocols can be fully served with ~80 bytes on BTC, and 223 bytes on BCH (`MAX_OP_RETURN_RELAY`). I'll update this PR to set it to 223 for now.
 
(I'm continuing to sync a node with a 1MB limit right now anyway, so I'll review after it's done.)